lib/characters.js

```diff
@@ -0,0 +1,6 @@
+export async function createFetchCharacters() {
```

Comment:
maybe rename the function to `fetchCharacters`because we're not really creating anything here :)
```suggestion
export async function fetchCharacters() {
```